apolloschurchapp/schema.graphql

```diff
@@ -671,6 +671,12 @@ type SmsPinResult {
   userAuthStatus: USER_AUTH_STATUS
 }
 
+type SocialMediaInfo {
```

Comment:
@redreceipt Is this relevant to the PR or just a sneak-in?

Reply:
not relevant to anything honestly, in the current version of core, that file is .gitignored because it's generated on deploy